firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/SignInResultActivity.java

```diff
@@ -25,6 +25,8 @@
 public class SignInResultActivity extends AppCompatActivity {
   private static final String TAG = "SignInResultActivity:";
 
+  public SignInResultActivity() {}
```

Comment:
Do we need this? Seems like the class is public already, so the default constructor will be public too

Reply:
This was a left over from trying to make it package-private will remove

Reply:
I'm pretty sure even if the class is marked as public, the constructor won't be public unless it's also marked as such.

Reply:
good to know! thanks!